builder/vmware/common/driver_fusion6.go

```diff
@@ -124,6 +124,25 @@ func (d *Fusion6Driver) Verify() error {
 	return compareVersions(matches[1], VMWARE_FUSION_VERSION, "Fusion Professional")
 }
 
+func (d *Fusion6Driver) ToolsIsoPath(k string) string {
+	// Fusion 13.x.x changes tools iso location
+	vmxpath := filepath.Join(d.AppPath, "Contents", "Library", "vmware-vmx")
+	var stderr bytes.Buffer
+	cmd := exec.Command(vmxpath, "-v")
+	cmd.Stderr = &stderr
+	versionRe := regexp.MustCompile(`(?i)VMware [a-z0-9-]+ (\d+)\.`)
+	matches := versionRe.FindStringSubmatch(stderr.String())
+	if matches[1] >= "13" {
+		if runtime.GOOS == "windows" && runtime.GOARCH == "arm" {
```

Comment:
Pardon me if I'm misunderstanding something, but isn't VMWare Fusion for macOS only? If so, this GOOS check will always return false, won't it?
Also if the subdirectories only concern the arch, the check should only be on `runtime.GOARCH`

Reply:
Also in terms of the possible values for `GOARCH`, there are several possibilities for ARM, according to https://github.com/golang/go/blob/master/src/go/build/syslist.go#L59, `arm` is one of those, but if you're running a modern macOS I'd think this is more likely to be `arm64`.

Reply:
> Pardon me if I'm misunderstanding something, but isn't VMWare Fusion for macOS only?

Yes, VMware Fusion (and VMware Fusion Player) is only available for macOS, whereas VMware Workstation (and VMware Workstation Player) is available for both Windows and Linux.

Reply:
That's a good point didn't think that one through all the way. Need to determine guest OS and cpu arch would match host so the arch is correct. How do we determine guest OS flavor. Currently only windows has a seperate Arm iso for tools.
![image](https://user-images.githubusercontent.com/3476391/214440763-b7b23932-39bd-4db1-b37c-a98db08c15b1.png)

Reply:
> Also in terms of the possible values for `GOARCH`, there are several possibilities for ARM, according to https://github.com/golang/go/blob/master/src/go/build/syslist.go#L59, `arm` is one of those, but if you're running a modern macOS I'd think this is more likely to be `arm64`.

You are correct I was thinking of the guest OS windows in this case as it's the only 1 vmware fusion has arm iso tools for.